### ISSUE TITLE
deprecated?

### ISSUE THREAD

**Issue by fanatid** [cryptocoinjs/sha256#5]
Probably should be marked as deprecated, because [crypto-browserify/createHash](https://github.com/crypto-browserify/createHash/)

**Comment by dcousens** [comment]
Indeed,  I think it should marked more clearly, not just in the README.
Thoughts @jprichardson?
